From 7f64f3624a9257516f6226dd000d8efa81c1adb3 Mon Sep 17 00:00:00 2001
From: Takashi Sakamoto <o-takashi@sakamocchi.jp>
Date: Sun, 24 Sep 2023 12:14:25 +0900
Subject: [PATCH] hwdb: ieee1394-unit-function: add Miglia Technology Harmony
 Audio (HA02)

This commit adds hwdb entry for Miglia Technology Harmony Audio (HA02).
The device is an application of OXford Semiconductor FW970 and will be
supported by ALSA oxfw driver in future.

(cherry picked from commit fb86af639922fdf0b4aac6660e456bda6538b738)

Related: RHEL-5950
---
 hwdb.d/80-ieee1394-unit-function.hwdb | 10 ++++++++++
 1 file changed, 10 insertions(+)

diff --git a/hwdb.d/80-ieee1394-unit-function.hwdb b/hwdb.d/80-ieee1394-unit-function.hwdb
index d62a2dec4b..e49e78f218 100644
--- a/hwdb.d/80-ieee1394-unit-function.hwdb
+++ b/hwdb.d/80-ieee1394-unit-function.hwdb
@@ -629,6 +629,16 @@ ieee1394:ven0000022Emo00800007sp0000A02Dver00010001
  IEEE1394_UNIT_FUNCTION_AUDIO=1
  IEEE1394_UNIT_FUNCTION_VIDEO=0
 
+# Please take an extra care since the vendor ID (0x0030E0) is for Oxford Semiconductor Ltd. and the
+# model ID (0x00f970) is commonly in the other OXFW970 models.
+ieee1394:node:ven0x0030e0mo0x00f970units0x00a02d:0x010001
+ieee1394:ven000030E0mo0000F970sp0000A02Dver00010001
+ ID_VENDOR_FROM_DATABASE=Miglia Technology
+ ID_MODEL_FROM_DATABASE=Harmony Audio
+ IEEE1394_UNIT_FUNCTION_MIDI=0
+ IEEE1394_UNIT_FUNCTION_AUDIO=1
+ IEEE1394_UNIT_FUNCTION_VIDEO=0
+
 #
 # TC Applied Technologies. Digital Interface Communication Engine (DICE).
 #
